packages/webapp/src/pages/Connection/Create.tsx

```diff
@@ -463,21 +463,28 @@ export default function IntegrationCreate() {
 
         if (integration?.authMode === 'JWT') {
             const credentials: string[] = [];
-            if (privateKeyId) {
-                credentials.push(`privateKeyId: '${privateKeyId}'`);
-            }
-            if (issuerId) {
-                credentials.push(`issuerId: '${issuerId}'`);
-            }
-            if (privateKey) {
-                credentials.push(`privateKey: '${privateKey}'`);
+
+            if (integration.provider.includes('ghost-admin')) {
+                const [id = '', secret = ''] = privateKey.split(':');
+                credentials.push(`privateKey: { id: '${id}', secret: '${secret}' }`);
+            } else {
+                if (privateKeyId) {
+                    credentials.push(`privateKeyId: '${privateKeyId}'`);
+                }
+                if (issuerId) {
+                    credentials.push(`issuerId: '${issuerId}'`);
+                }
+                if (privateKey) {
+                    credentials.push(`privateKey: '${privateKey}'`);
+                }
             }
+
             if (credentials.length > 0) {
                 jwtCredentialsString = `
-                credentials: {
-                    ${credentials.join(',\n            ')}
-                }
-                `;
+    credentials: {
+        ${credentials.join(',\n            ')}
+    }
+    `;
```

Comment:
formatting is wrong here right?

Reply:
yes, formatting was initially wrong, with `ghost-admin`  we need to split the `apiKey` to `id` and `secret` and at the webapp

Reply:
no I mean the tabulations change of the 4 lines above has been committed by accident

Reply:
it's a template string for display

Reply:
yes, for indentation.